tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -162,14 +177,16 @@ Task StartAsync()
                 AddShutdownTask();
 
                 _probeStatusPoller.StartPolling();
-                _symbolsUploader.StartExtractingAssemblySymbolsAsync();
-                return _debuggerSink.StartFlushingAsync();
+                _symbolsUploader.StartFlushingAsync();
+                _diagnosticsUploader.StartFlushingAsync();
+                return _snapshotUploader.StartFlushingAsync();
             }
 
             void AddShutdownTask()
             {
                 LifetimeManager.Instance.AddShutdownTask(() => _discoveryService.RemoveSubscription(DiscoveryCallback));
-                LifetimeManager.Instance.AddShutdownTask(_debuggerSink.Dispose);
+                LifetimeManager.Instance.AddShutdownTask(_snapshotUploader.Dispose);
+                LifetimeManager.Instance.AddShutdownTask(_diagnosticsUploader.Dispose);
```

Comment:
If remember this properly, the intention was to ensure that if any disposal task fails, it will not compromise the execution or completion of other tasks. 🤔

Reply:
I mean, a disposal task _shouldn't_ fail IMO 😅 If it _could_ we should put the contents in a try-catch?

Reply:
Hmmm, I agree! Made a relevant change here d23ac6ab7992b51a305225e1483f185a30667be0